contexts/security-v3-unstable.jsonld

```diff
@@ -2,12 +2,53 @@
   "@context": [{
     "id": "@id",
     "type": "@type",
-    "@protected": true,
     "JsonWebKey2020": {
       "@id": "https://w3id.org/security#JsonWebKey2020"
     },
     "JsonWebSignature2020": {
-      "@id": "https://w3id.org/security#JsonWebSignature2020"
+      "@id": "https://w3id.org/security#JsonWebSignature2020",
+      "@context": {
+        "@version": 1.1,
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "challenge": "https://w3id.org/security#challenge",
+        "created": {
+          "@id": "http://purl.org/dc/terms/created",
+          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+        },
+        "domain": "https://w3id.org/security#domain",
+        "expires": {
+          "@id": "https://w3id.org/security#expiration",
+          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+        },
+        "jws": "https://w3id.org/security#jws",
+        "nonce": "https://w3id.org/security#nonce",
+        "proofPurpose": {
```

Comment:
This is going to need to match the definition below or this will cause a redefinition conflict

Reply:
thank you for pointing this out.